addons/skin.confluence/720p/AddonBrowser.xml

```diff
@@ -108,6 +97,11 @@
 					<include>ButtonCommonValues</include>
 					<label>24034</label>
 				</control>
+				<control type="button" id="5">
+					<description>Settings foreign</description>
+					<include>ButtonCommonValues</include>
+					<label>5</label>
```

Comment:
could you use string id 10004 to get rid of low number usage? Is the description correct?